db/migrate/20240619123052_create_web_hook_events_daily_aggregates.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateWebHookEventsDailyAggregates < ActiveRecord::Migration[7.0]
+  def change
+    create_table :web_hook_events_daily_aggregates do |t|
+      t.belongs_to :web_hook, null: false, index: true
```

Comment:
woah I haven't seen this -- I thought in a migration it was either add `t.integer` or `t.references` aight aight. Maybe I'm getting older and forgetting. Documentation on this looks dead on

Reply:
Yeah I had to double check this too, never seen it. But yeah it's correct and seems to do a similar thing to `t.references` :)